app/components/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import Axios from 'axios';
+import PropTypes from 'react-proptypes';
+import MovieContainer from './MovieContainer';
+import MovieTrailerPlayer from './MovieTrailerPlayer';
+
+class App extends Component {
+  state = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    const { config } = this.props;
+    Axios.get(config.API_URL).then(({ data = [] }) => {
+      this.setState({ movies: data });
+    });
+  }
+
+  render() {
+    const { movies } = this.state;
+    const { config } = this.props;
+
+    return (
+      <div>
```

Comment:
You could use React fragment instead of returning a div: https://reactjs.org/docs/fragments.html